### DISCUSSION TITLE
Convert to outside collaborator

### DISCUSSION THREAD

**Discussion by illnerqd** [community/community#108376] [Enterprise]
Hello everybody, I have been trying to **convert a member of an organisation to an outside collaborator**. Both times the member was removed from Member category but not added to the section 'Outside collaborators'. This is odd, maybe an internal bug? 

Another issue is that this member's co-authored commits disappeared from their public profile as they contributed to a private repo in an organisation. (I guess that is according to the internal configuration of Github). But the members's contribution should be shown in their profile. That is the reason to keep them as an outside collaborator. Thanks for investigating this.

**Comment by kyuna0312** [comment]
did you added that contributers in your  group. Collab should be member of your Organization

**Reply by illnerqd** [reply]
Thanks for the fast reply! Yes, first the user was added as a member, after that I used the 'convert to outside collaborator...' option. (shown in the image below). 
But all that happened was that the member was removed from the organisation and no outside collaborator was added. 

![Screenshot from 2024-02-15 16-57-55](https://github.com/community/community/assets/156677925/77db3f79-e7c0-4ee8-811b-46f5cbd34cd0)

**Comment by benoittgt** [comment]
Message say that the member should have direct access to a repo but even if it is the case the user is removed from the organisation. 

<img width="625" height="259" alt="Screenshot 2026-03-17 at 11 27 25" src="https://github.com/user-attachments/assets/335d940c-999a-4787-95c2-99ad80db0dd9" />

For me this is a bug.
